### ISSUE TITLE
Is there a way to force the widget to have a fixed height?

### ISSUE THREAD

**Issue by jshelley** [Ophidian-AC/charcomponent#8]

**Comment by Ophidian-AC** [comment]
Absolutely, you mean so the second page doesn't seem smaller compared to the rest?

**Comment by jshelley** [comment]
Yeah, exactly. Especially used for placing this component next to other components on page and not wanting to have content below it moved as the user progresses through the screens.

**Comment by Ophidian-AC** [comment]
Definitely! I'll make them all the same height as the first page, would that work?

**Comment by jshelley** [comment]
Is there any chance you can make that configurable? I can see if this is
used in a modal it might be ok for the screens to resize.

**Comment by Ophidian-AC** [comment]
Hmm...I could make it configurable in the sense that you can determine how tall the overall widget is, but not to switch the uniform height on/off,...I will double check just in case, but I dont think so!

**Comment by jshelley** [comment]
Ok, no problem. If it can't be configured via the configuration object, go
ahead and make all pages the same size as the first page. If you can
document how I can change it in code in the wiki that would be appreciated.

**Comment by Ophidian-AC** [comment]
Will do! Working on wrapping everything up so I can write the docs on how to install and setup as well, I apologize for taking a little longer on that, just working to keep the file sizes to a minimum so it's nice and speedy

**Comment by jshelley** [comment]
No worries! Thanks for all of the hard work!

**Comment by Ophidian-AC** [comment]
Anytime! Did you have a hard deadline that you absolutely need it by? So I can be aware of how much time I have to refactor and finish up

**Comment by jshelley** [comment]
It would be good to have things wrapped up by Sunday evening, but if you
need a bit more time mid next week is fine.

**Comment by Ophidian-AC** [comment]
Ok, I will do my best!

**Comment by jshelley** [comment]
No worries if you need more time. You are doing a great job so I don't want
to rush you in finishing things up!
